src/handlers/mod.rs

```diff
@@ -374,3 +378,49 @@ impl ForeignToplevelHandler for State {
     }
 }
 delegate_foreign_toplevel!(State);
+
+impl DrmLeaseHandler for State {
+    fn drm_lease_state(&mut self, node: DrmNode) -> &mut DrmLeaseState {
```

Comment:
All these methods are guaranteed to receive a `node` that exists in our `OutputDevice`? Looking at those unwraps.

Reply:
The Smithay docs do not make this very clear but I would be inclined to believe so. Looking at the Smithay source, it seems that the only nodes that are used are created from the node used when creating a DrmLeaseState, based on [the global creation step](https://github.com/Smithay/smithay/blob/91e61f13501f21d66803efac947bfafed43080c5/src/wayland/drm_lease/mod.rs#L428), [the data initialization step](https://github.com/Smithay/smithay/blob/91e61f13501f21d66803efac947bfafed43080c5/src/wayland/drm_lease/mod.rs#L714), and the [request creation](https://github.com/Smithay/smithay/blob/91e61f13501f21d66803efac947bfafed43080c5/src/wayland/drm_lease/mod.rs#L762).